dime-coding-standards/checklists/tex/sampling_randomization_checklist.tex

```diff
@@ -0,0 +1,190 @@
+[]% after the title body
+
+\newcommand{\dimeCheckBox}[1]{\CheckBox[height=0.01cm, width=0.4cm, bordercolor=gray]{#1}}
+\newcommand{\dimeTextField}[3]{\TextField[name=#1, height=0.3cm, width=#2, bordercolor=gray]{#3}}
+
+
+\newcommand{\titleBox}[1]{
+	\begin{tcolorbox}
+		[colframe = fontcolor,
+		colback = fontcolor,
+		sharp corners,
+		halign = flush center,
+		valign = center,
+		height = 0.3\textwidth,
+		after skip = 1cm]
+		#1
+	\end{tcolorbox}
+}
+
+\begin{document}
+	\begin{fullwidth}
+		
+		\titleBox{
+			\textcolor{white}{\LARGE{\textbf{DIME Analytics \\ Peer Code Review - Sampling and Randomization}} \\
+				\Large\textbf{{v1.0}}}
+		}
+
+  \dimeTextField{time}{2.5 cm}{Reviewer:} 
+
+        \dimeTextField{time}{2.5 cm} {Coder:}
+		
+		\textbf{NOTE:} Make sure that fill out this checklist \textbf{ONLY IF} your partner's submission includes    \textbf{sampling and/or randomization} tasks. 
+		
+		\section*{Sampling and Randomization tasks}
+  This checklist lists important factors to consider while reviewing your code review partner's \textbf{sampling/randomization} code. Please fill this checklist, and submit it as an attachment when you submit \href{https://survey.wb.surveycto.com/collect/code_review_summary?caseid=}{this detailed form.}
+  \hfill \break 
+  \break
+  \textbf{Note:} We are piloting this checklist as part of improving the code review process. The questions in this list are therefore \textbf{not required}, and we welcome your feedback on the points listed below.
+
+  
+		\begin{itemize}
+			\setlength\itemsep{-0.1em}
+   
+   
+    \item[]{\textbf{Sampling:}
+   
+            \begin{itemize}
+            
+            \item[]\dimeCheckBox{\textbf{The code clearly identifies the population eligible for sampling}. (This should be in the form of an original dataset that is fixed, unless sampling is continuous or ongoing due to research design; in that case, this should be clearly described)}
+         
+            
+            \begin{itemize}
+                \item[]\dimeCheckBox{The population dataset is uniquely identified.}
+                
+                \item[]\dimeCheckBox{The population dataset contains all variables used in the sampling process, such as clusters and strata.}
+                
+                \item[]\dimeCheckBox{\textbf{Advanced check:} Clusters, strata, and individual-level IDs do not contain unmasked information about the identity of the clusters, strata, or individuals.} 
+
+                \item[] \dimeCheckBox{\textbf{Advanced check:} The original dataset should be checks for stability when loaded, using some combination of commands like \texttt{datasignature}, file hashing, \texttt{assert}, and/or \texttt{isid, sort} (or an equivalent in other coding languages.)}
+                
+            \end{itemize}
+            
+            
+			\item[] \dimeCheckBox{\textbf{The method and rationale for random processes is described in documentation and understandable in code}, including:} 
```

Comment:
Compare to my comment on the HFC file about not having check box on the title. It is unclear to me what should be required for this box to be checked.